### PR TITLE
add <jbroot> environment variable

### PR DESCRIPTION
**Pull request by tuancc-repo**
just a stub, and keep /var/jb compatible.

### CODE REVIEW

**Comment by tuancc-repo** [comment]
Add support for jailbroken apps/tweaks to access files in /var/jb after hide-environment (temporarily remove /var/jb path symbols)

**Comment by mkwiimaster7** [comment]
I hope this gets added to Dopamine lol

**Comment by bakirfatah99** [comment]
Hello opa334 next update will add rootfull jailbreak?

I would appreciate it if you replied!

File: BaseBin/systemhook/src/main.c [hunk 1]
```diff
@@ -331,6 +332,13 @@ __attribute__((constructor)) static void initializer(void)
 	unsandbox();
 	loadExecutablePath();
 
+	/* Add support for jailbroken apps/tweaks to access files in /var/jb
+		 after hide-environment (temporarily remove /var/jb path symbols) */
+	char jbroot[PATH_MAX]={0};
+	if(readlink("/var/jb", jbroot, sizeof(jbroot)) > 0) {
+		setenv("jbroot", jbroot, 0);
```
Comment (Cryptiiiic) [inline BaseBin/systemhook/src/main.c:339]: env vars are usually standardized as UPPERCASE so JBROOT

**Review by Cryptiiiic — CHANGES_REQUESTED** [review]

**Pull request closed without merging**